src/social-network/defaults/createDataWithIdentifierChangedListener.ts

```diff
@@ -24,6 +25,8 @@ export function createDataWithIdentifierChangedListener<
                 throw new Error('Invalid state')
             }
         }
-        ref.value = next
+        if (!isEqual(ref.value, next)) {
+            ref.value = next
```

Comment:
I think there should've already been one in `ValueRef`... need confirmation by @Jack-Works

Reply:
It's true. Revoked!